src/routes/albums.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const albumsController = require('../controllers/albums');
+const router = express.Router();
+
+
+router.post('/:id/albums', albumsController.createAlbum);
+
+router.get('/', albumsController.readAlbum);
+
+router.get('/:id', albumsController.getAlbumById);
+
+router.patch('/:id', albumsController.updateAlbum);
+
+router.delete('/:id', albumsController.deleteAlbum);
+
+module.exports = router;
```

Comment:
These routes could all be grouped together. Also, you never need more than one consecutive blank line to separate your code:
```
const router = express.Router();

router.post('/:id/albums', albumsController.createAlbum);

router.get('/', albumsController.readAlbum);
router.get('/:id', albumsController.getAlbumById);
router.patch('/:id', albumsController.updateAlbum);
router.delete('/:id', albumsController.deleteAlbum);

module.exports = router;
```